//
//
// Copyright (c) 2015, University of Michigan.
// All rights reserved.
//
// This source code is licensed under the BSD-style license found in the
// LICENSE file in the root directory of this source tree. An additional grant
// of patent rights can be found in the PATENTS file in the same directory.
//
//

//
// TODO:
//
// @author: Johann Hauswald
// @contact: jahausw@umich.edu
//

package kp;

message all_kp {
    required uint32 size = 1;
    repeated Keypoint keypoint = 2;
}


message Keypoint {
    required float x = 1;
    required float y = 2;
    required float size  = 3;
    required float angle = 4;
    required float response = 5;
    required int32 octave = 6;
    required int32 class_id = 7;
}
